corpus_transformation_scripts/Elexicon/Begriffe_eLexikon.xml

```diff
@@ -1,74 +1,108 @@
 	<lem elex="arrha">arra</lem>
 	<lem elex="auctor">auctor</lem>
-	<lem elex="auctor">auctor</lem>
 	<lem elex="auctoritas">auctoritas</lem>
+	<lem elex="auditores">auditor</lem>
 	<lem elex="auditores">auditores</lem>
 	<lem elex="basilica">basilica</lem>
+	<lem elex="basilica">Basilica_Ortsname</lem>
 	<lem elex="benedictio">benedictio</lem>
+	<lem elex="benedictio">benedictus</lem>
+	<lem elex="benedictio">benedicere</lem>
+	<lem elex="benedictio">benedico</lem>
 	<lem elex="beneficium">beneficium</lem>
 	<lem elex="boni_homines">boni homines</lem>
 	<lem elex="boni_homines">bonus homo</lem>
-	<lem elex="burgundi">burgundi</lem>
-	<lem elex="burgundi">burgundio</lem>
+	<lem elex="burgundi">Burgundia</lem>
+	<lem elex="burgundi">Burgundio</lem>
 	<lem elex="burgus">burgus</lem>
-	<lem elex="caespitaticum">caespitaticum</lem>
+	<lem elex="burgus">Burgus_Ortsname</lem>
+	<lem elex="caespitaticum">caespitaticus</lem>
 	<lem elex="camerarius">camerarius</lem>
+	<lem elex="camerarius">camerlanus</lem>
+	<lem elex="camerarius">camerlengus</lem>
+	<lem elex="camerarius">Camerlengus_Personenname</lem>
+	<lem elex="camerarius">subcamerarius</lem>
 	<lem elex="cancellarius">cancellarius</lem>
 	<lem elex="canonicus">canonicus</lem>
+	<lem elex="canonicus">cocanonicus</lem>
 	<lem elex="caritas">caritas</lem>
+	<lem elex="caritas">caritativus</lem>
 	<lem elex="carropera">carropera</lem>
-	<lem elex="carta">carta</lem>
+	<lem elex="carropera">carroperae</lem>
 	<lem elex="carta">charta</lem>
+	<lem elex="carta">chartula</lem>
+	<lem elex="carta">carta</lem>
+	<lem elex="carta">cartula</lem>
+	<lem elex="carta">cartola</lem>
 	<lem elex="cathedra">cathedra</lem>
 	<lem elex="causa">causa</lem>
+	<lem elex="causa">causella</lem>
 	<lem elex="cautio">cautio</lem>
 	<lem elex="cella">cella</lem>
+	<lem elex="cella">Cella_Ortsname</lem>
 	<lem elex="cellula">cellula</lem>
 	<lem elex="census">census</lem>
 	<lem elex="cessio">cessio</lem>
 	<!--<lem elex="centenarius">centenarius</lem>-->
-	<lem elex="chrisam">chrisam</lem>
+	<lem elex="chrisam">chrisma</lem>
 	<lem elex="civis">civis</lem>
 	<lem elex="civitas">civitas</lem>
 	<lem elex="clementia">clementia</lem>
+	<lem elex="clementia">clemens</lem>
+	<lem elex="clementia">Clemens_Personenname</lem>
+	<lem elex="clementia">clementer</lem>
 	<lem elex="clericus">clericus</lem>
+	<lem elex="clericus">clericusque</lem>
 	<lem elex="cognatio">cognatio</lem>
+	<lem elex="cognatio">cognatus</lem>
 	<lem elex="colonus">colonus</lem>
 	<lem elex="comes">comes</lem>
 	<lem elex="comes_palatinus">comes palatinus</lem>
 	<!--<lem elex="commendatio">commendare</lem>-->
 	<!--<lem elex="commendatio">commendatio</lem>-->
 	<lem elex="communio">communio</lem>
 	<lem elex="commutatio">commutatio</lem>
-	<lem elex="commutatio">conmutatio</lem>
+	<lem elex="commutatio">commutatioque</lem>
 	<lem elex="compositio">compositio</lem>
 	<lem elex="condita">condita</lem>
-	<lem elex="confirmatio">confirmare</lem>
```

Comment:
Ich hatte hier "confirmare" und "confirmatio" in alphabetische Reihefolge gemacht. Das macht es einfacher die Termini in der Datei zu finden.